web/src/app/main/components/ToolbarTitle.js

```diff
@@ -32,6 +32,10 @@ const styles = {
   title: {
     padding: '0 4px 0 4px',
     flex: 1, // pushes toolbar actions to the right
+    fontWeight: 500,
+    lineHeight: 1.6,
+    fontSize: '1.25rem',
+    letterSpacing: '0.0075em',
```

Comment:
same here with the weight, height, and spacing, unless needed we should fall back to MUI's default